classes/Customization.php

```diff
@@ -287,6 +287,14 @@ public static function retrieveQuantitiesFromIds($idsCustomizations)
      */
     public static function countQuantityByCart($idCart)
     {
+        @trigger_error(
+            sprintf(
+                '%s is deprecated since version 8.0.0. This method is no longer used to validate order cancellation.',
```

Comment:
I'm not sure of this deprecated message. I think that this method can be called for something else that order cancellation validation 🤔

Reply:
Yes, you are right. I will remove this deprecation.

Reply:
Resolved in https://github.com/PrestaShop/PrestaShop/pull/12422/commits/facb0227b0dfcdbdd84d5e4aa9c373a448d24e4a